packages/forms/resources/views/components/textarea.blade.php

```diff
@@ -13,9 +13,10 @@
             ax-load-src="{{ \Filament\Support\Facades\FilamentAsset::getAlpineComponentSrc('textarea', 'filament/forms') }}"
             x-data="textareaFormComponent({ initialHeight: @js($initialHeight) })"
             x-ignore
-            x-on:load.window="render()"
+            x-init="render()"
```

Comment:
Could you move this to `init()` inside the JS file maybe?

Reply:
Done in commit https://github.com/filamentphp/filament/pull/8193/commits/ef67b59ea81097a7345ba20d356c1e83a5a8d702